.github/ISSUE_TEMPLATE/docs-report--quasar-v1.yml

```
@@ -9,17 +9,17 @@ body:
       value: |
         ### Read Before Proceeding Further
 
-        This form is for submitting a **docs report** for Quasar Framework(**v1**). (https://v1.quasar.dev)
-        So, if you are using another version, want to report a bug, just seeking help, or want to submit a feature request, please return back and choose a more suitable option from the list. 
+        This form is for submitting a **docs report** for Quasar Framework(**v1**). (https://v1.quasar.dev).
```

```suggestion
        This form is for submitting a **docs report** for Quasar Framework(**v1**). (https://v1.quasar.dev)
```
Is the last dot really needed as it's not a sentence?